docs/codeql/ql-training/query-examples/java/query-injection-java-1.ql

```diff
@@ -4,4 +4,4 @@ from Method m, MethodAccess ma
 where
   m.getName().matches("sparql%Query") and
   ma.getMethod() = m
-select ma, m
\ No newline at end of file
+select ma, m
```

Comment:
This was probably unintentional? (I assume this is also why the documentation label was added)

Reply:
I'm perplexed :smile: